post_installation_configuration/index.adoc

```diff
@@ -63,23 +63,27 @@ As a cluster administrator, you can perform the following operations with the ma
 ** Enable Device Manager
 
 * xref:../post_installation_configuration/network-configuration.adoc#post-install-network-configuration[Configure network]:
-After installing {product-title}, as a cluster administrator, you can configure the following:
+After installing {product-title}, you can configure the following:
 
 ** Ingress cluster traffic
 ** Node port service range
 ** Network policy
 ** Enabling the cluster-wide proxy
 
 * xref:../post_installation_configuration/storage-configuration.adoc#post-install-storage-configuration[Configure storage]:
-By default, containers operate using ephemeral storage or transient local storage. The ephemeral storage has a lifetime limitation, so you must configure persistent storage to store the data for a long time.
+By default, containers operate using ephemeral storage or transient local storage. The ephemeral storage has a lifetime limitation. TO store the data for a long time, you must configure persistent storage.
 You can configure storage by using one of the following methods:
 
 ** *Dynamic provisioning*: You can dynamically provision storage on demand by defining and creating storage classes that control different levels of storage, including storage access.
 
-** *Static provisioning*: Cluster administrators can use Kubernetes persistent volumes to make existing storage available to a cluster by supporting various device configurations and mount options.
+** *Static provisioning*: You can use Kubernetes persistent volumes to make existing storage available to a cluster. Static provisioning can support various device configurations and mount options.
 
 * xref:../post_installation_configuration/preparing-for-users.adoc#post-install-preparing-for-users[Configure users]:
-OAuth access tokens allow users to authenticate themselves to the API. As a cluster administrator, you can configure OAuth to specify an identity provider, use role-based access control to define and apply permissions to users, and install an Operator from OperatorHub.
+OAuth access tokens allow users to authenticate themselves to the API. As a cluster administrator, you can configure OAuth to perform the following tasks:
+
+* Specify an identity provider
```

Comment:
Please add + so it renders as part of the sentence

Reply:
otherwise looks good , nice content improvement

Reply:
acked